amazonredshift_blueprints/upload_file.py

```diff
@@ -125,7 +114,8 @@ def combine_folder_and_file_name(folder_name, file_name):
 
     return combined_name
 
-def map_datatypes(pandas_type:str) -> str:
+
+def map_datatypes(pandas_type: str) -> str:
```

Comment:
Since you're getting rid of the `create_table_query` function, you should remove this too since that is the only place where it is used